### ISSUE TITLE
Not getting Email in auth raw_info

### ISSUE THREAD

**Issue by vishnun** [arunagw/omniauth-twitter#97]
I want to get the email address as well when I ask the user to sign in with Twitter.

I see the latest commit being related to that. But it's not been released. As of now 1.2.0 is the latest gem from April 22 and this commit is more recent. 

I would like to request you to release the latest version with the email support for twitter.

**Comment by tommybstitch** [comment]
I was under the impression you couldn't get the user's email address from the Twitter API.

**Comment by guilhermesimoes** [comment]
Now you can. You need your application to be whitelisted though. https://dev.twitter.com/rest/reference/get/account/verify_credentials

**Comment by ahmadajmi** [comment]
Yes the latest commit didn't yet included in a release, for me I did

```
gem 'omniauth-twitter', git: 'https://github.com/arunagw/omniauth-twitter.git'
```

Then do a request to access the email address from [Twitter API Support](https://support.twitter.com/forms/platform) and choose the `I need access to special authentication privileges (e.g. xAuth)` option then `I need xAuth for my application` and then fill the form with your app info.

I noticed that the user should verify his email address on twitter before i can access it.
